AudioLink/Shaders/HackyExport.shader

```diff
@@ -0,0 +1,112 @@
+﻿Shader "Unlit/HackyExport"
+{
+    Properties
+    {
+        _MainTex ("Texture", 2D) = "white" {}
+    }
+        SubShader
+    {
+        Tags { "RenderType" = "Opaque" }
+        LOD 100
+
+        Pass
+        {
+            Cull Off
+
+            CGPROGRAM
+            #pragma vertex vert
+            #pragma fragment frag
+            // make fog work
+            #pragma multi_compile_fog
+
+            #include "UnityCG.cginc"
+
+            struct appdata
+            {
+                float4 vertex : POSITION;
+                float2 uv : TEXCOORD0;
+            };
+
+            struct v2f
+            {
+                float2 uv : TEXCOORD0;
+                UNITY_FOG_COORDS(1)
+                float4 vertex : SV_POSITION;
+            };
+
+            sampler2D _MainTex;
+            float4 _MainTex_ST;
+
+            v2f vert (appdata v)
+            {
+                v2f o;
+                o.vertex = UnityObjectToClipPos(v.vertex);
+                o.uv = TRANSFORM_TEX(v.uv, _MainTex);
+                UNITY_TRANSFER_FOG(o,o.vertex);
+                return o;
+            }
+
+            fixed4 frag (v2f i) : SV_Target
+            {
+                // sample the texture
+                fixed4 col = tex2D(_MainTex, i.uv);
+                // apply fog
+                UNITY_APPLY_FOG(i.fogCoord, col);
+                col = 1.;
+                return col;
+            }
+            ENDCG
+        }
+        GrabPass
+        {
+            "_AudioTexture"
```

Comment:
If the goal is of this shader is just to export some texture globally, one can use the camera trick we already use for the main AudioLink prefab like I mention above.

Reply:
With URP, this ended up being a dead end.

Reply:
Dead end as in this file should not be included, or dead end as in the existing functionality didn't work?